parsec_web/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 2-Clause License
```

Comment:
wrong license

Reply:
Done! The BSD 2-Clause License is `equations-parser` came from C++ library.